embedded/sql/sql_tx.go

```diff
@@ -142,6 +142,6 @@ func (sqlTx *SQLTx) Closed() bool {
 	return sqlTx.closed
 }
 
-func (sqlTx *SQLTx) delete(key []byte) error {
+func (sqlTx *SQLTx) Delete(key []byte) error {
```

Comment:
I wouldn't expose this method, sql statement should not expose the low level key

Reply:
resolved